bridges/TelegramBridge.php

```diff
@@ -16,9 +16,12 @@ class TelegramBridge extends BridgeAbstract
         ]
     ];
     const TEST_DETECT_PARAMETERS = [
-        'https://t.me/s/durov' => ['username' => 'durov'],
-        'https://t.me/durov' => ['username' => 'durov'],
-        'http://t.me/durov' => ['username' => 'durov'],
+        'https://t.me/s/rssbridge' => ['username' => 'rssbridge'],
```

Comment:
Is this channel controlled by RSS-Bridge project? Might be nice to add a comment to the code otherwise.